data/data/agent/files/usr/local/bin/start-assisted-service.sh.template

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+MIRROR_IMAGES_MOUNT=""
+
+if [ -f  '{{.MirrorRegistriesFilename}}' ]; then
```

Comment:
What about using a template directly for the `assisted-service.service` file, and using instead the template conditionals to optionally mount the registies.conf? In that way this file will not be required

Reply:
Good point. Changed to use template for the service and removed script.